build.gradle

```diff
@@ -0,0 +1,74 @@
+plugins {
+    id 'maven-publish'
+    id 'net.researchgate.release' version '2.3.4'
+    id "com.jfrog.bintray" version "1.7"
+    id 'java-gradle-plugin'
+}
+
+repositories {
+    jcenter()
+}
+
+def actualVersion = "$version" // remapping version to this var, since version has namespace clashes in some cases (e.g. within bintray.pkg.version)
+def scmTag='v'+actualVersion
+
+group = 'org.testeditor.fixture'
+version = actualVersion
+
+jar {
+    manifest {
+        attributes 'version': actualVersion
+    }
+}
+
+// for more configuration options see https://github.com/bintray/gradle-bintray-plugin
+bintray {
+    user = System.getenv()['JCENTER_USER'] 
+    key = System.getenv()['JCENTER_PASSWD']
+    publications = ['FixturePublication'] 
+    publish = true 
+    pkg {
+        repo = 'Fixtures'
```

Comment:
Should we really maintain different Maven repositories? We could pack everything into `test-editor-maven`.